### PR TITLE
Update package url for 1.4 iOS install script

### LINKED ISSUE #129: Update Couchbase Lite version for tutorial app
- Update the Couchbase Lite version to 1.4 in the training application.

### PR DESCRIPTION
**Pull request by sethrosetter**
Update iOS install to 1.4 by default

Fixes #129 #130

### CODE REVIEW

File: android/app/src/main/java/com/couchbase/todo/TasksFragment.java [hunk 1]
```diff
@@ -253,7 +253,8 @@ public TaskAdapter(Context context, LiveQuery query) {
             if (thumbnail != null)
                 imageView.setImageBitmap(thumbnail);
             else
-                imageView.setImageDrawable(getResources().getDrawable(R.drawable.ic_camera_light));
+                //imageView.setImageDrawable(getResources().getDrawable(R.drawable.ic_camera_light));
+                imageView.setImageDrawable(getResources().getDrawable(R.drawable.ic_note));
```
Comment (jamiltz) [inline android/app/src/main/java/com/couchbase/todo/TasksFragment.java]: @hideki is this change intended?

**Comment by sethrosetter** [comment]
@jamiltz. Please let @ssmotra know if you merge this as his demo is has been pointing at this branch.

**Comment by jamiltz** [comment]
I think that we should merge it and deploy the changes that relate to updating the deps to 1.4.

But there are two things that we should do before:

1. The convergence changes are 1.5 so we won't have anything on developer.couchbase.com just yet but ideally there should be a lesson for this topic (probably all of it can be the content from @ssmotra's demo). So ideally that change (the config file) should be on another PR. Would it be possible to do that?

2. And i'm just waiting to see if that change from @hideki is intended.

**Comment by jamiltz** [comment]
Oh but there won't be a CBL 1.5. Point 1 is fine then, can be in this PR.

Just need to check point 2 above and then I'll merge.

File: android/app/src/main/java/com/couchbase/todo/TasksFragment.java (hunk 1 above)
Comment (hideki) [inline android/app/src/main/java/com/couchbase/todo/TasksFragment.java]: Sachin would like to change default icon of the task to note icon instead of the camera icon.

**Comment by jamiltz** [comment]
ok point 2 was because that branch wasn't rebased (that change is already on master). Merging 👍
